modules/ROOT/pages/cross_cutting/test_automation/architecture_tests.adoc

```diff
@@ -0,0 +1,56 @@
+= Architecture Tests
```

Comment:
When we are talking about Architecture tests maybe we should also mention the possibility to add custom linting rules to ensure coding guidelines, even if I would not really consider them to be tests.

Reply:
Is this really a practice that we recommend for Java projects? I know that we have done custom Sonar rules in some projects. But all of them can now be done easier in ArchUnit without having the dependency to SonarQube versions.

Reply:
You are right, most probably this is more a thing for typescript projects where you can do much more stuff you should not do compared to Java. Was just a thought while reading your article